rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderTest.java

```diff
@@ -3598,4 +3496,4 @@ private FederationStorageProvider createFederationStorageProvider(Repository rep
     private BridgeStorageProvider createBridgeStorageProvider(Repository repository, NetworkParameters networkParameters, ActivationConfig.ForBlock activations) {
         return new BridgeStorageProvider(repository, bridgeAddress, networkParameters, activations);
     }
-}
+}
```

Comment:
Why remove this blank space? Creates a warning from github

Reply:
Updated.